### DISCUSSION TITLE
请问用shizuku开启无障碍权限的命令是什么？

### DISCUSSION THREAD

**Discussion by JoffreyN** [gkd-kit/gkd#828] [Q&A] [answered]
比如类似这样的 shizuku('settings put secure enabled_accessibility_services org.autojs.autojs6/org.autojs.autojs.core.accessibility.AccessibilityServiceUsher');

对于此应用，该如何写这个命令？

**Comment by lisonge** [comment, marked as answer]
```js
shizuku('settings put secure enabled_accessibility_services li.songe.gkd/.service.A11yService');
```

**Reply by JoffreyN** [reply]
谢谢(#^.^#)🙏🏻这是autojs6提供的方法。
